cunumeric/coverage.py

```diff
@@ -97,7 +99,14 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
         def wrapper(*args: Any, **kwargs: Any) -> Any:
             return func(*args, **kwargs)
 
-    wrapper._cunumeric = CuWrapperMetadata(implemented=True)
+    # TODO (bev) Scraping text to set flags seems a bit fragile. It would be
+    # preferable to start with flags, and use those to update docstrings.
+    multi = "Multiple GPUs" in (getattr(func, "__doc__", None) or "")
+    single = "Single GPU" in (getattr(func, "__doc__", None) or "") or multi
+
+    wrapper._cunumeric = CuWrapperMetadata(
+        implemented=True, single=single, multi=multi
+    )
```

Comment:
I'm still not quite in love with this. Either for the text scraping, or the separate single/multi flags. Multi always implies single, correct? In that case an enum might be more appropriate.

Reply:
> Multi always implies single, correct?

Yes, in no case do we have multi-GPU support without single-GPU support.

Reply:
OK I will plan to update both (flags -> docstrings, and an enum for single/multi spec) together in a later PR.